### DISCUSSION TITLE
wrong output when trying BraTS20

### DISCUSSION THREAD

**Discussion by Mark-bit3** [WuJunde/MedSegDiff#172] [Q&A]
Hi, I have tried to run the given script following instructions but get an segmentation of the whole brain rather than the ground truth tumor. Is there some suggestion when facing this problem?

**Comment by Mark-bit3** [comment]
I have also try to train your model on ISIC skin dataset, the performance is also not ideal even after 50000 steps.
![image](https://github.com/KidsWithTokens/MedSegDiff/assets/160187330/7bad8185-60b9-418b-93ab-d27b100d2e15)
![image](https://github.com/KidsWithTokens/MedSegDiff/assets/160187330/8491cc30-6376-48a4-9f92-15fcb05dc4bf)

**Comment by doqizuo** [comment]
> Hi, I have tried to run the given script following instructions but get an segmentation of the whole brain rather than the ground truth tumor. Is there some suggestion when facing this problem?

hi! bro~,I am running the given script following instructions,but I am facing one problem after another.May I ask you for advice?

**Comment by Tai-Yng** [comment]
Hello, have you solved it now? Why is the segmented part a whole piece?

**Comment by abdullah22842** [comment]
I trained Brats2021 but I am getting wrong output> 
<img width="202" height="1530" alt="image" src="https://github.com/user-attachments/assets/82a7d3f2-1b8d-44e2-aeae-0bfd7f047af8" />
and evaluation for 3D Brats is different than 2D ISIC. Can anyone here, having same issue.
